shared/src/testFixtures/kotlin/com/egm/stellio/shared/ParameterizedTests.kt

```diff
@@ -0,0 +1,74 @@
+package com.egm.stellio.shared
+
+import com.egm.stellio.shared.util.loadSampleData
+import org.junit.jupiter.params.provider.Arguments
+import java.time.OffsetDateTime
+import java.util.stream.Stream
+
+class ParameterizedTests {
+
+    companion object {
+        fun rawResultsProvider(): Stream<Arguments> {
```

Comment:
The returned data is specific to the search service, there is no interest to put this class into the shared module.